public/app/features/templating/variable_srv.ts

```diff
@@ -65,12 +67,27 @@ export class VariableSrv {
         this.templateSrv.setGlobalVariable('__org', {
           value: {
             name: contextSrv.user.orgName,
-            id: contextSrv.user.id,
+            id: contextSrv.user.orgId,
             toString: function() {
               return this.id;
             },
           },
         });
+        getBackendSrv()
+          .get('/api/users/' + contextSrv.user.id)
+          .then((user: User) => {
+            this.templateSrv.setGlobalVariable('__user', {
+              value: {
+                name: user.name,
+                login: user.login,
+                email: user.email,
+                id: contextSrv.user.id,
+                toString: function() {
+                  return this.id;
+                },
+              },
+            });
+          });
```

Comment:
Do we really need to do this call? Isn't all this information already available in GrafanaBootData? ping @torkelo

Reply:
@hugohaggmark 

I have changed so this is all pulled from contextSrv  (and added email/name to contextSrv).   

My original thinking here was to expand this to include the ability to pull info from a variety of API structures.   For now - being able to use the users login name/email in a query allows a dataset to be reduced to only those rows related to the individual user.   (In essence allowing a personalised dashboard)

The user information is available to the client post login both via an API, and in BootData.   One assumes that an administrator will not create users that have a name/email etc that are so structured that they permit SQL/code injection.